vendor/bat-native-ads/src/bat/ads/internal/confirmations/confirmations_state.cc

```diff
@@ -213,9 +213,11 @@ std::string ConfirmationsState::ToJson() {
       base::Value(std::move(failed_confirmations)));
 
   // Ad rewards
-  base::Value ad_rewards = ad_rewards_->GetAsDictionary();
-  dictionary.SetKey("ads_rewards",
-      base::Value(std::move(ad_rewards)));
+  if (ad_rewards_) {
```

Comment:
required for tests that use confirmations state that is not part of an integration test. https://github.com/brave/brave-browser/issues/12563 will remove ad rewards from confirmations